subprojects/gradle/cd/src/main/kotlin/com/avito/ci/VerifyOutputsTask.kt

```diff
@@ -43,13 +43,15 @@ abstract class VerifyOutputsTask @Inject constructor(objects: ObjectFactory) : D
             outputsDir = outputsDir
         )
 
+        val logger = GradleLoggerFactory.fromTask(this).create("VerifyOutputsTask")
```

Comment:
It's so common pattern when you simply want to create a logger without reusing the factory.
Could we create fun `GradleLoggerFactory.getLogger(task)`?

Reply:
yep, added `inline fun <reified T : Task> getLogger(task: T): Logger = fromTask(task).create<T>()`

Reply:
also created same method for plugins